index.html

```diff
@@ -61,9 +62,203 @@ <h3>LET'S CONNECT</h3>
                             </li>
                         </ul>
                     </div>
+               
+                    <!-- start work section -->
+                    
+                    <div class="work-section">
```

Comment:
- [ ] Please kindly replace the `div` tag with a `section` tag for semantic reasons.